src/components/ListItem.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import styled from "styled-components";
+
+const Container = styled.li`
+  padding: 18px;
+  border-bottom: 1px solid #dbdbdb;
+`;
+
+const Thumbnail = styled.img`
+  width: 100%;
+  height: 144px;
+  object-fit: cover;
+`;
+
+const Info = styled.div`
+  padding: 12px 7px 16px 0px;
+`;
+
+const Title = styled.span`
+  font-size: 1.2em;
+`;
+
+const Score = styled.span`
+  color: #ff792a;
+  font-size: 1.3em;
+  margin-left: 8px;
+`;
+
+const Address = styled.p`
+  font-size: 0.75rem;
+  margin-top: 8px;
+  color: #9b9b9b;
+`;
+
+const Review = styled.p`
+  margin-top: 10px;
+  font-size: 0.875rem;
+`;
+
+export default function ListItem(place) {
+  return (
+    <Container key={place.name}>
+      <Thumbnail src={place.photo} />
+      <Info>
+        <Title>{place.name}</Title>
+        <Score>{place.score}/5</Score>
+        <Address>{place.address}</Address>
+        <Review>{place.comment}</Review>
+      </Info>
+    </Container>
+  );
```

Comment:
```suggestion
export default function ListItem({ place }) {
  return (
    <Container key={place.name}>
      <Thumbnail src={place.photo} />
      <Info>
        <Title>{place.name}</Title>
        <Score>{place.score}/5</Score>
        <Address>{place.address}</Address>
        <Review>{place.comment}</Review>
      </Info>
    </Container>
  );
```

컴포넌트라면 [구조분해할당](https://ko.javascript.info/destructuring-assignment)을 통해 props를 받아야합니다.
저 상태라면 그냥 함수지 컴포넌트가 아녜요.